### DISCUSSION TITLE
Coding style

### DISCUSSION THREAD

**Discussion by alfredbuehler** [creecros/Mailmagik#15] [General]
May I suggest, as soon we have a Makefile, to add a tool like [PHP-CS-Fixer](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer), to establish a consistent coding style. (indentation, braces, spaces etc.)

What do you think? Is it too much?

**Comment by creecros** [comment]
it's fine with me, I'm probably the world's worst when I type code, so it's probably useful.

**Comment by creecros** [comment]
and it looks simple enough to run, ill try and do it tonight

**Comment by creecros** [comment]
https://github.com/creecros/kbphpimap/commit/359189bbcb4a88dff3578a999fc0535f52fc63d4

**Reply by alfredbuehler** [reply]
Great, thank you! But can we add it to the project as dev-dependency, to run via Makefile?

**Reply by creecros** [reply]
oh, ya thats fine too, if i told you I had google what a makefile is....

**Reply by creecros** [reply]
learn something new everyday

**Comment by creecros** [comment]
@alfredbuehler Added the vendor library, but ill let you do the makefile, i dont know how

**Reply by alfredbuehler** [reply]
Now it's getting serious.
```
abu@abu-tc ~/prj/Mailmagik (main) $ make  test-php-style-fix
vendor/bin/php-cs-fixer fix --diff --allow-risky yes .
Composer detected issues in your platform:

Your Composer dependencies require a PHP version ">= 8.1.0". You are running 7.4.33.
```
- I have installed php-cs-fixer locally at /usr/local/bin, so that I can use it in every project.
- You have added it a normal dependency, but it is only required or recommender for development, under "require-dev"  in composer.json.

In this situation, it makes maybe no sense to add php-cs-fixer to the project. I vote for removing. 

The Makefile is adjusted accordingly.
Sorry for the inconvenience.

**Reply by creecros** [reply]
lol, you guys and you old php versions!
